### ISSUE TITLE
"permission denied" when running cloudprober/cloudprober:v0.10.7

### ISSUE THREAD

**Issue by drigz** [google/cloudprober#378]
I get an error when using the new Docker image:

```
docker run -it cloudprober/cloudprober:v0.10.7
docker: Error response from daemon: OCI runtime create failed: container_linux.go:346: starting container process caused "exec: \"/cloudprober\": permission denied": unknown.
```

0.10.6 works fine when started the same way.

**Comment by manugarg** [comment]
@drigz Thank you so much for the report. I indeed changed the workflow that was being used to create the image. Makefile was the same, but the source of the cloudprober binary got changed -- instead of building a new one right before packaging, I was using Github action to download the binary built in the previous workflow. I'll fix the workflows, but in the meantime I've fixed the docker image. It should work now.

01:26:54 ~ :) docker run -it cloudprober/cloudprober:v0.10.7
W0228 09:26:56.546895       1 cloudprober.go:139] Config file /etc/cloudprober.cfg not found. Using default config.
01:26:59 ~ :) docker run -it cloudprober/cloudprober:v0.10.7 --version
v0.10.7
